### DISCUSSION TITLE
Yomogyへの記事の投稿手順

### DISCUSSION THREAD

**Discussion by y0m0gy** [yomogyhub/yomogy_test#35] [General]
手順:

1. Github アカウント作成: まず、Github のアカウントを作成してください。その後、コミュニティメンバーとしての申請を行ってください。承認後に記事の投稿などが可能となります。
2. 著者情報の登録: post/all-author.jsonに従って、自身の著者情報を追加または修正してください。必要に応じて、public/author/{任意}.pngに 200px × 200px の画像を配置してください。(形式に注意してください。エラーが出た場合、先に記事を作成してください。)
3. 記事の作成: blog-templateからテンプレートをダウンロードし、指示に従って記事を作成してください。
4. 記事の投稿: 作成した記事はdevブランチのpost/blog/{希望のカテゴリー}配下に追加してください。
5. 画像の追加: 画像はdevブランチに投稿することで自動的に更新されます。もし、トップ画像の変更が必要であれば、public/images/blog/{カテゴリー}に 1200px × 630px の画像を配置してください。

**Comment by takashi-yamanashi** [comment]
コミュニティメンバーとしての申請フォームはこちらです
https://forms.gle/tCuE8HzymknGH24R7
